### PR TITLE
Allow max concurrent jobs definition

### PR DESCRIPTION
**Pull request by pbouchez**
- Do not enforce `Maximum Concurrent Jobs`

### CODE REVIEW

**Review by milaflq — DISMISSED** [review]

**Comment by btravouillon** [comment]
See https://github.com/mila-iqia/ansible-role-bareos/pull/34#discussion_r1572249487 for variables naming.

File: templates/myself.conf.j2 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # {{ ansible_managed }}
 Client {
   Name = {{ item.name }}
-  Maximum Concurrent Jobs = 20
+  Maximum Concurrent Jobs = {{ bareos_client_maximum_concurrent_jobs | default('20') }}
```
Comment (btravouillon) [inline templates/myself.conf.j2]: This may forbid to define different `Maximum Concurrent Jobs` for different clients.

Comment (pbouchez) [reply templates/myself.conf.j2]: Fixed

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**